javascriptv3/example_code/lambda/README.md

```diff
@@ -1,30 +1,63 @@
-# AWS Lambda JavaScript SDK v3 code examples
-AWS Lambda lets you run code without provisioning or managing servers. You pay only for the compute time you consume.
+# AWS Lambda code examples for the AWS SDK for JavaScript in Node.js
+
+## Overview
+
+Shows how to use the SDK for JavaScript in Node.js with AWS Lambda (Lambda) to manage and invoke
```

Comment:
You don't have to put (Lambda) after the full name unless it's substantially different (like RDS or S3).

Reply:
Makes sense. Fixed.